### PR TITLE
Crews documentation

### PR DESCRIPTION
**Pull request by williamhogman**
WIP. Please let me know if I missed something

### CODE REVIEW

File: crews.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Klira Crews
+
+The basic division the team is into Crews. Each Crew has a captain who
+is responsible for facilitating the operational working routines of
+each crew. The crew's decide for themselves how they want to structure
+their work.
+
+The captain of each crew is responsible for operational procedures of
+their crew. Each crew is decides on what routines and processes they
```
Comment (mantono) [inline crews.md]: >Each crew is decides on

No "_is_" maybe?

File: crews.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+want they use.
```
Comment (mantono) [inline crews.md]: Replace "they" with "to".

File: crews.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Additionally the crew should also maintain:
+
+- A backlog with tasks to be completed
+- A kanban board with all the work in progress.
+- A weekly report with much time the team was able to spend working
+  towards their primary goals.
+
+## Voyages
+
+Each quarter the crew should together with their manager plan their
```
Comment (mantono) [inline crews.md]: Who is the _manager_? The CTO? Or the crew's captain?

File: crews.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+A crew should at least have the following routines:
+
+- A daily stand-up style meeting
+- A weekly planning meeting
+
+Additionally the crew should also maintain:
+
+- A backlog with tasks to be completed
+- A kanban board with all the work in progress.
+- A weekly report with much time the team was able to spend working
```
Comment (mantono) [inline crews.md]: >with _how_ much time

Also, I would like to know a little bit more about this.
- How should the reporting be done?
- How detailed should the reports be?

**Comment by Torbacka** [comment]
Would like to add a port where different crew can share and exchange treasures. A.k.a demo.

File: crews.md (hunk 4 above)
Comment (Torbacka) [reply crews.md]: I would also like to know who the report is for.

Comment (williamhogman) [reply crews.md]: The report is for me and Pontus

**Comment by williamhogman** [comment]
I like the idea I will add a demo section

**Comment by williamhogman** [comment]
Made some changes regarding the report. Changing it to monthly and done and monthly granularity. Exactly the report should be done is up to each crew.

File: crews.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+- Objective: "Integrate with more banks catering to high-risk customers"
+- Key result: "Deploy 3 new bank-integrations"
+
+### The Pirate's Cove
+
+Every four weeks all the crews meet up to show off their loot (code),
+share stories (learnings), sing sea shanties and eat and drink (enjoy
+themselves). The captain of each crew summarizes what the crew has
+been doing before allowing the team members to demo their features.
```
Comment (mantono) [inline crews.md]: I like this! This seems like a good (and fun) way to share knowledge and team progress.

File: crews.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+- A daily stand-up style meeting
+- A weekly planning meeting
+
+Additionally the crew should also maintain:
+
+- A backlog with tasks to be completed
+- A kanban board with all the work in progress.
+- A monthly report with how many the team was able to spend working
+  towards their primary goals versus how much time was spent doing other things.
```
Comment (mantono) [inline crews.md]: I think we should state more exact what is expected from the monthly report, or we risk spending too much time (or not enough time) on the reports. For example
- Should it summarize time spent for team as whole or individual team members
- How granular should they be? Hourly, days, half-days? Or just how many percentage that was spent on maintenance/legacy vs new projects/objective

Maybe an example report for a team would be good?

File: crews.md (hunk 3 above)
Comment (zinob) [reply crews.md]: That would generally be the team-manager. In this case @williamhogman

File: crews.md (hunk 6 above)
Comment (williamhogman) [reply crews.md]: added an example but it was deliberately vague so as to not influence you too much.

**Pull request merged**